kinetic/distribution.yaml

```diff
@@ -5044,6 +5044,14 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2016.4.24-0
     status: maintained
+  libmynteye-release:
+    release:
+      packages:
+      - libmynteye
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/harjeb/libmynteye-release.git
+      version: 0.0.1-0
```

Comment:
I see that you don't have a `doc` or `source` section for this package.  Neither is required, but they can give more information to your users (`doc` will generate documentation for hosting on the wiki, and `source` will allow the build-farm to build the package from source, and do pull request testing).  There's some more information about Pull request testing here http://wiki.ros.org/buildfarm/Pull%20request%20testing

Reply:
OK, I will close this pull request,  and rerun bloom